Sources/IRGen/Expression/IRExpression.swift

```diff
@@ -8,6 +8,20 @@
 import AST
 import Lexer
 
+struct GeneratedCode {
+  var preamble: String
+  var expression: String
+
+  init(_ preamble: String, _ expression: String) {
+    self.preamble = preamble
+    self.expression = expression
+  }
+
+  func render() -> String {
```

Comment:
In-line with the previous codegen implementation for consistency I think this should be called `rendered()`